aws-logs-destination/src/main/java/software/amazon/logs/destination/UpdateHandler.java

```diff
@@ -0,0 +1,61 @@
+package software.amazon.logs.destination;
+
+import software.amazon.awssdk.services.cloudwatchlogs.CloudWatchLogsClient;
+import software.amazon.cloudformation.Action;
+import software.amazon.cloudformation.exceptions.CfnNotFoundException;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.HandlerErrorCode;
+import software.amazon.cloudformation.proxy.Logger;
+import software.amazon.cloudformation.proxy.OperationStatus;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.ProxyClient;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+
+public class UpdateHandler extends BaseHandlerStd {
+
+    private static final String DESTINATION_UPDATE_GRAPH = "AWS-Logs-Destination::Update";
+    private static final String DESTINATION_POLICY_UPDATE_GRAPH = "AWS-Logs-DestinationPolicy::Update";
+
+    protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(final AmazonWebServicesClientProxy proxy,
+            final ResourceHandlerRequest<ResourceModel> request, final CallbackContext callbackContext,
+            final ProxyClient<CloudWatchLogsClient> proxyClient, final Logger logger) {
+
+        final ResourceModel model = request.getDesiredResourceState();
+        final ResourceModel previousModel = request.getPreviousResourceState();
+
+        return ProgressEvent.progress(model, callbackContext)
+                .then(progress -> {
+                    if (! isUpdatable(model, previousModel)) {
+                        return ProgressEvent.<ResourceModel, CallbackContext>builder().errorCode(
+                                HandlerErrorCode.NotUpdatable)
+                                .status(OperationStatus.FAILED)
+                                .build();
+                    }
```

Comment:
have u tested it locally w/o this check? this might not be needed

Reply:
As discussed offline, I have removed this check